agm_minutes/2015AGM_minutes.md

```diff
@@ -0,0 +1,115 @@
+Travel costs for the meeting will be covered by the Society.
+
+As such, Zeth said that he would like to focus on a bid to bring EuroPython to the UK.
+He then proposed that Peter be "General Secretary", to be "responsible for crew and communications", and that Daniele Procida be "Local Chair", responsible for arrangements with the venue.
+
+Zeth then discussed how the conference has been planned for this year, including mentioning the use of GitHub issues.
+He also mentioned that the conference website is built using Python 2.
+Russel Winder has volunteered to update the conference website to use Python 3.
+
+Zeth then thanked Nick, Peter, Mary, Sarah, Luke, Naomi, and others who had worked to put the conference on, and asked for a standing ovation for Nick, who is standing down from the committee this year and who has been responsible for the success of the education track.
+
+Zeth said the conference was planned, the conference was organised, and the conference was done.
+
+## Treasurer's report
+
+Mary presented the income and expenditure accounts for the 2014 conference, which can be found at https://github.com/PyconUK/PyConUKSociety/blob/master/agm_finances/2014AGM_finances.md
+
+For 2014 we made a profit of around £5000.
+
+Mary also reported that around £1700 has been raised for Cancer Research so far at the conference, and that there's a bucket at reception that people can use for further donations.
+
+At this point, Nick thanked Mary and asked for a standing ovation.
+
+## Proposal for financial support plan
+
+Zeth asked Nick to present a new financial support plan.
+
+Nick proposed that we use some of the profits from this year's conference to provide financial support to delegates for next year.
+
+This was voted for by a show of hands.
+
+## Statement on diversity
+
+When people bought tickets, they were asked for their gender.
+Zeth showed a graph of the responses, which showed that 20% of delegates identify as female, 72% of delegates identify as male, 2% as non-binary, with the remainder not answering the question.
+
+Zeth also said that we had had many more applications to DjangoGirls than could be accepted.
+
+## Statement on education
+
+Nicholas gave a brief account of the education track and said that he felt it had been successful.
+
+## Statement on co-operation with the PSF
+
+Zeth said that because of the size of conference budget, the PyCon UK Society will need to become a charity, and that we will work with Van and the PSF to do this.
+
+## Proposal to bring back unconference
+
+Zeth reminded the meeting that in previous years there has been a UK Python unconference, funded by tips.
+A show of hands showed that people were interested in the idea.
+
+## Next UK conference
+
+Zeth said that the venue has been a great place to hold a conference, and that in particular the staff have been very good to work with.
+However, the conference has outgrown the venue and we will have to find a new venue for 2016.
```

Comment:
May not want to emphasise this point. Just in case some detail of Cardiff does not pan out (though I am sure it will). Bizarrely people loved the overflows, I thought we would get lynched.